README.md

```diff
@@ -131,6 +131,9 @@ Note: Upstart/SysV init based OS types are not supported.
   - [kube-router](https://github.com/cloudnativelabs/kube-router) v1.0.0
   - [multus](https://github.com/intel/multus-cni) v3.4.2
   - [weave](https://github.com/weaveworks/weave) v2.6.5
+- Ingress
+  - [ambassador](https://www.getambassador.io/): 1.5
+  - [nginx](https://kubernetes.github.io/ingress-nginx): 0.32.0
```

Comment:
There is already a mention of nginx-ingress in the "Application" section a few lines below. Can move the ambassador mention there?

All other links points to the GitHub repo, so I think it would be fair to do the same for Ambassador.